CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+# Contributing to Caliptra
+
+## CLA
+
+Per the [Charter](Caliptra_WG_Technical_Charter.md), contributors to Caliptra
```

Comment:
Is this enforceable through Github? Or will maintainers check that the contributor, or their company, has signed the CLA, before the maintainer will accept the contribution?

Reply:
Not enforceable to github, must be manually verified before merging (however, the list of CLA signers is public)